joomla/joomla-cms
# [4.0] Fix Error in configuration of com_content.

Pull Request for Issue #19543 

### Summary of Changes
Switcher element accepts radio type field.
So, type changed to the radio.


### Testing Instructions
1.Open configuration settings of <code>com_content</code> Or <code>Articles</code>
2.Then open <code>list layout</code> tab.
3.Check on the bottom no error present. If you have Debugger installed then you can easily see.

But without debugger just use browser inspect.


### Expected result
![screenshot from 2018-02-02 23-28-03](https://user-images.githubusercontent.com/30978328/35759317-c7adfb4a-0870-11e8-9424-969822efd446.png)



### Actual result
![screenshot from 2018-02-02 23-02-59](https://user-images.githubusercontent.com/30978328/35759319-caaaf776-0870-11e8-8069-a10b70feb313.png)
